doc/source/building_containers.rst

```diff
@@ -0,0 +1,130 @@
+           <expose>
+               <port number="80"/>
+               <port number="8080"/>
+           </expose>
+           <volumes>
+               <volume name="/tmp"/>
+               <volume name="/var/log"/>
+           </volumes>
+           <environment>
+               <env name="PATH" value="/bin:/usr/bin:/home/user/bin"/>
+               <env name="SOMEVAR" value="somevalue"/>
+           </environment>
+           <labels>
+               <label name="somelabel" value="labelvalue"/>
+               <label name="someotherlabel" value="anotherlabelvalue"/>
+           </labels>
+       </containerconfig>
+   </type>
+
+Other DockerFile directives like **ADD**, **RUN** and **COPY** are
+covered by KIWI by other mechanisms, since they aren't metadata, they are
+part of the image :ref:`prepare step <prepare-step>` (install packages,
+include config files, arrange configurations, etc.). Files can be
+included using the :ref:`overlay tree (directory) <image-components>`
+and any configuration script can be included as part of the
+:ref:`config.sh <custom-config.sh>` file. See a complete container
+example `here <https://github.com/SUSE/kiwi-descriptions>`_.
+
+.. tip:: Load Docker images
+
+   KIWI images can be loaded to the Docker daemon by just running:
+
+   .. code-block:: bash
+
+      $ docker load -i my_image_file.tar.xz
+
+Building derived images
+-----------------------
+
+KIWI also supports making a container image on top of another base image.
+In that case, the resulting image will include the base image layers
+plus a new one containing the changes added by KIWI. Building derived
+images works in the same way as for the base images, the only difference
+is that the base image must be specified in the description file, it can
+be done using the **derived_from** optional attribute of ``<type>`` tag.
+The value of the **derived_from** attribute is the URI of the image;
+currently KIWI only supports references to local files (it expects a xz
+compressed tarball image) and any other URI type that is supported by
+the :command:skopeo tool (e.g. DockerHub URIs as
+``derived_from="docker://opensuse:leap"``). The type section to make a
+derived image from an openSUSE Leap base image could be:
+
+.. code:: xml
+
+   <type image="docker" derived_from="docker://opensuse:leap">
+       <containerconfig name="container_name" tag="container_tag" maintainer="tux">
+           <entrypoint execute="myscript.sh"/>
+           <subcommand clear="true"/>
+       </containerconfig>
+   </type>
+
+It is also relevant to note here the usage of ``<subcommand clear="true">``.
+It forces the cleanup of any ``subcommand`` inherited from the base image.
+Note that the configuration metadata is inhereted from the base image to
+the derived one, the only way to change the inhereted metadata is by
+overwriting it. Only ``entrypoint`` and ``subcommand`` are an exception
+to that and they can be cleared using the **clear** attribute.
```

Comment:
Better use double backticks around `clear` to make it consistent with the other tags.